pkg/logs/input/container/launcher.go

```diff
@@ -37,14 +40,14 @@ func NewLauncher(collectAll bool, collectFromFiles bool, sources *config.LogSour
 		}
 		log.Infof("Could not setup the kubernetes launcher: %v", err)
 
-		launcher, err = docker.NewLauncher(sources, services, pipelineProvider, registry, false)
+		launcher, err = docker.NewLauncher(sources, services, pipelineProvider, registry, false, dockerReadTimeout)
```

Comment:
nit: for readability, it'd be nice to keep the order of the arguments identical to `container.NewLauncher()`

Reply:
@olivielpeau Done but not sure I'm happy with the result (the timeout becomes the first parameter). I can change the `container.NewLauncher()` instead but it already has a logical one. It's not that important I'm going with what you suggest.

Reply:
yeah that's fair, could've changed `container.NewLauncher` instead... Not very important, if we keep adding options we should probably just create a `launcherOptions` struct